pkg/controller/cassandracluster/generator.go

```diff
@@ -752,6 +752,15 @@ func createCassandraContainer(cc *api.CassandraCluster, status *api.CassandraClu
 		},
 
 		Lifecycle: &v1.Lifecycle{
+		    PostStart: &v1.Handler{
+				Exec: &v1.ExecAction{
+					Command: []string{
+						"/bin/bash",
+						"-c",
+						"/etc/cassandra/cassandra_post_start.sh",
```

Comment:
Your changes would make any deployment that doesn't provide that script fail. Your PostStart action should check if the file exists and if it does should run.

Also it doesn't mean that cassandra was started but just that the container was started as it's mentioned in the documentation _"There is no guarantee, however, that the postStart handler is called before the Container's entrypoint is called"_
So I think there is still some work missing here

Reply:
@ajaybhat @SuhailRahman 👆